### PR TITLE
Upgrade to autoextract-poet 0.3.0

### PR DESCRIPTION
**Pull request by ivanprado**

### CODE REVIEW

**Comment by ivanprado** [comment]
@kmike I added you in case you have any comments. My plan is to release tomorrow. I can address any comment that comes after the merge in a separate PR anyway.

File: README.rst [hunk 1]
```diff
@@ -267,3 +281,4 @@ When using the AutoExtract providers, be aware that:
 .. _`Scrapy's asyncio documentation`: https://docs.scrapy.org/en/latest/topics/asyncio.html
 .. _`Request-level error`: https://doc.scrapinghub.com/autoextract.html#request-level
 .. _`Query-level error`: https://doc.scrapinghub.com/autoextract.html#query-level
+.. _`supported page types`: https://autoextract-poet.readthedocs.io/en/more_page_types/_autosummary/autoextract_poet.pages.html
```
Comment (kmike) [inline README.rst:284]: I think we should use either latest or stable branch in this link, no more_page_types

File: CHANGES.rst [hunk 2]
```diff
@@ -1,6 +1,14 @@
 Changes
 =======
 
+0.7.0 (2021-08-04)
```
Comment (kmike) [inline CHANGES.rst:4]: if you're making release tomorrow, date is going to be different :)

**Review by kmike — APPROVED** [review]
👍

File: README.rst (hunk 1 above)
Comment (ivanprado) [reply README.rst]: Good catch. Fixed in https://github.com/scrapinghub/scrapy-autoextract/pull/25/commits/380a377953f137cf2c4f60bcd4b8c6ce61b68428. I see that `_autosummary` is also part of the URL, this is ugly. I would have chosen a different name if I would have known it. Something to be changed in the future.

**Comment by ivanprado** [comment]
Thank you @kmike for the quick review :-) Merging to make a release

**Pull request merged**